TwilioVerifySDK/TwilioVerify/Sources/Models/VerifyFactorPayload.swift

```diff
@@ -23,19 +23,25 @@ import Foundation
 public protocol VerifyFactorPayload {
   /// Factor sid
   var sid: String { get }
+  var allowIphoneMigration: Bool { get }
 }
 
 ///Describes the information required to verify a **Factor** which type is **Push**
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/c51c4513007d818dab4b64cd25c837959c7ec03d/TwilioVerifySDK/TwilioVerify/Sources/Models/VerifyFactorPayload.swift#L29"></span>Prefer at least one space after slashes for comments.<br />
<code>comment_spacing</code> <code>VerifyFactorPayload.swift:29</code></td>
    </tr>
  </tbody>
</table>